plugins/modules/source_control/gitlab/gitlab_group_members.py

```diff
@@ -0,0 +1,235 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Zainab Alsaffar <Zainab.Alsaffar@mail.rit.edu>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: gitlab_group_members
+short_description: Manage group members on GitLab Server
+description:
+    - This module allows to add and remove members to/from a group, or change a member's access level in a group on GitLab.
+version_added: '1.2.0'
+author: Zainab Alsaffar (@zanssa)
+requirements:
+    - python-gitlab python module <= 1.15.0
+    - administrator rights on the GitLab server
+extends_documentation_fragment: community.general.auth_basic
+options:
+    api_token:
+        description:
+            - A personal access token to authenticate with the GitLab API.
+        required: true
+        type: str
+    gitlab_group:
+        description:
+            - The name of the GitLab group the member is added to/removed from.
+        required: true
+        type: str
+    gitlab_user:
+        description:
+            - The username of the member to add to/remove from the GitLab group.
+        required: true
+        type: str
+    access_level:
+        description:
+            - The access level for the user.
+            - Required if I(state=present), user state is set to present.
```

Comment:
From the documentation, I would assume that this changes the access level of the user accordingly if the user is already present in the group, but the access level differs.

Reply:
@felixfontein, The documentation tells users that they have to enter a value for the access level when state is present. 

When state is absent to remove a member from the group, they wouldn't need to type in the access level for the member in order to be removed from the group.

Also, this doesn't update the access level of an existing member!